test_package_validation.py

```
@@ -0,0 +1,249 @@
+
+def test_credential_paths():
+    """Test 4: Verify credential path configuration"""
+    print("\n=== Test 4: Credential Path Configuration ===")
+    
+    try:
+        # Test that the default token cache path uses the correct directory
+        from zoho_mcp.config import settings
+        
+        # Check if the path contains the expected directory name
+        if "/.zoho-mcp/" in settings.TOKEN_CACHE_PATH:
+            print(f"✅ Token cache path uses ~/.zoho-mcp/ directory")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)